spec/include_directive/parser_spec.rb

```diff
@@ -28,6 +28,19 @@
     expect(hash).to eq expected
   end
 
+  it 'treats spaces as part of the resource name' do
+    str = 'friends, comments.author , posts.author,posts. comments.author'
+    hash = JSONAPI::IncludeDirective::Parser.parse_include_args(str)
+    expected = {
+      friends: {},
+      ' comments'.to_sym => { 'author '.to_sym => {} },
```

Comment:
`:' comments' => { ... }` might work as well. I don't really have an opinion on what's best. Plus it's totally not critical.